diff --git a/lib/command.js b/lib/command.js
index a176dfbb988815a43370a401b573047b1c58231d..b58444f08671346b48467abb3a0cff9d4da08075 100644
--- a/lib/command.js
+++ b/lib/command.js
@@ -732,6 +732,19 @@ Expecting one of '${allowedValues.join("', '")}'`);
     return this;
   }
 
+  /**
+   * Add multiple options.
+   *
+   * @param {Option[]} options
+   * @return {Command} `this` command for chaining
+   */
+  addOptions(...options) {
+    for (const option of options) {
+      this.addOption(option);
+    }
+    return this;
+  }
+
   /**
    * Internal implementation shared by .option() and .requiredOption()
    *
diff --git a/typings/index.d.ts b/typings/index.d.ts
index 65f08c9231ac3c3fa774c74198811ebcd641045a..0c58d21c43acddea50749bd4f0741c5fa31fac7e 100644
--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -705,6 +705,8 @@ export class Command {
    */
   addOption(option: Option): this;
 
+  addOptions(...options: Option[]): this;
+
   /**
    * Whether to store option values as properties on command object,
    * or store separately (specify false). In both cases the option values can be accessed using .opts().